src/.nuget/dir.targets

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+
+  <Target Name="VerifyVCRedist" BeforeTargets="GetSymbolPackageFiles" Condition="'$(_runtimeOSFamily)' == 'win'">
```

Comment:
`GetSymbolPackageFiles` -> `GetPackageFiles`

It's kind of weird (to me), to rely on the machine state for these vs picking them up from somewhere else, like how the apiset packages were handled.